### DISCUSSION TITLE
apt-get updated my HyperBian image - broken now. Please help!

### DISCUSSION THREAD

**Discussion by amaisano** [hyperion-project/hyperion.ng#1286] [Q&A] [answered]
I was impatient and tried updating to hyperion alpha 10 today on my rpi3, which is running HyperBian.

The update via command line went fine, but when I rebooted, I'm getting no lights, no web interface, and an error:

`Cannot mix incompatible Qt library (version 0x50701) with this library (version 0x50b03)`

Now I'm kicking myself for not making a config backup. Is there any way to retrieve my last known config?

Is there a way to force start the web interface?

**Comment by Lord-Grey** [comment, marked as answer]
@amaisano see here
https://github.com/hyperion-project/hyperion.ng/issues/1277

You need to uninstall alpha9.

> Is there any way to retrieve my last known config?

You find the config in the „.hyperion“ Directory of the Home Directory of the user you are running Hyperion under.

**Reply by amaisano** [reply]
I tried that but the same problem occurred. I tried looking for the config, but there was none. Just a `db` directory with some default configuration data.

I ended up going with a fresh HyperBian image and re-configuring my Lightberry lights. All good now.

**Reply by Lord-Grey** [reply]
>  Just a db directory
 
 Just to close the information loop. In the db directory there is hyperion's configuration data base...
 If that was empty you are looking at the wrong user. I suppose you were running under root (maybe without knowing) before.
 Then the last configuration is in root's home....
 
 Nevertheless, good to hear that it is working now!

**Reply by amaisano** [reply]
Damn, I _knew_ I should have checked the `root` folder. Oh well, it's back in working order and I'm making backups off-site now :)
